mod/beacon/block_store/pruner.go

```diff
@@ -17,17 +17,18 @@
 // EXPRESS OR IMPLIED, INCLUDING (WITHOUT LIMITATION) WARRANTIES OF
 // MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, AND
 // TITLE.
-
 package blockstore
 
-import asynctypes "github.com/berachain/beacon-kit/mod/async/pkg/types"
+import (
+	"github.com/berachain/beacon-kit/mod/primitives/pkg/events"
+)
 
 // BuildPruneRangeFn builds a function that returns the range of blocks to
 // prune.
 func BuildPruneRangeFn[BeaconBlockT BeaconBlock](
 	cfg Config,
-) func(*asynctypes.Event[BeaconBlockT]) (uint64, uint64) {
-	return func(event *asynctypes.Event[BeaconBlockT]) (uint64, uint64) {
+) func(events.Event[BeaconBlockT]) (uint64, uint64) {
```

Comment:
events.Event is typically frowned upon since.
`<x>s.<X>` bad import pattern

Reply:
yeah true. Do you think we should rename the package from primitives/events -> primitives/async, or just create an "asynctypes/async" alias every time we import